canonical-websites/www.ubuntu.com
# 2656: Move /about/about-ubuntu/governance to /community/governance

# Note
**Please merge #2685 first**

## Done

- Moved /about/about-ubuntu/governance to /community/governance
- Updated copy according to [copydoc](https://docs.google.com/document/d/11obb5eQNVwAD8auz2SLFqiZeTVpqbj8uph0jShsZvVA/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/community/governance
- Check that it matches the [copydoc](https://docs.google.com/document/d/11obb5eQNVwAD8auz2SLFqiZeTVpqbj8uph0jShsZvVA/edit#)
- Check that /about/about-ubuntu/governance redirects to /community/governance

## Issue / Card

Fixes #2656
